CHANGELOG-unreleased.md

```diff
@@ -13,6 +13,7 @@ the released changes.
 - Moved design matrix normalization code from `pint.fitter` to the new `pint.utils.normalize_designmatrix()` function.
 - Made `Residuals` independent of `GLSFitter` (GLS chi2 is now computed using the new function `Residuals._calc_gls_chi2()`).
 ### Added
+- CHI2, CHI2R, TRES, DMRES now in par files
```

Comment:
Is this in all par files or only in pst-fit par files?

Reply:
only post-fit.  I will clarify